packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -152,11 +153,17 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       crawlerOverview: `${ENTERPRISE_SEARCH_DOCS}crawler.html`,
       deployTrainedModels: `${MACHINE_LEARNING_DOCS}ml-nlp-deploy-models.html`,
       documentLevelSecurity: `${ELASTICSEARCH_DOCS}document-level-security.html`,
-      elser: `${MACHINE_LEARNING_DOCS}ml-nlp-elser.html`,
+      elser: `${ENTERPRISE_SEARCH_DOCS}elser-text-expansion.html`,
```

Comment:
I'm intentionally changing the existing `elser` doc link to point to https://www.elastic.co/guide/en/enterprise-search/current/elser-text-expansion.html instead of https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html . This affects one existing component, the "Learn more" link in the ELSER 1-click deployment panel.

![Screenshot 2023-06-19 at 11 33 35](https://github.com/elastic/kibana/assets/14224983/6592386a-031c-4e4e-81af-313d85123b9a)

I think this change is fine because the Enterprise Seach ELSER documentation is more closely tied to the 1-click deployment panel. Let me know if I should revert the change.

Reply:
Sounds OK to me if you think this makes the most sense